src/pages/[platform]/build-a-backend/auth/auth-migration-guide/index.mdx

```diff
@@ -0,0 +1,436 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Migrate from v5 to v6',
+  description: '',
+  platforms: [
+    'angular',
+    'javascript',
+    'nextjs',
+    'react',
+    'vue'
+  ]
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+## Auth.signUp
+
+```diff
+-  import { Auth } from 'aws-amplify';
++  import { signUp } from 'aws-amplify/auth';
+
+-  Auth.signUp: (params: string | SignUpParams,  ...restOfAttrs: string[]) => Promise<ISignUpResult>;
++  signUp: (input: SignUpInput) => Promise<SignUpOutput>;
+```
+
+## Auth.confirmSignUp
+
+```diff
+-  import { Auth } from 'aws-amplify';
++  import { confirmSignUp } from 'aws-amplify/auth';
+
+-  Auth.confirmSignUp: (username: string, code: string, options?: ConfirmSignUpOptions) => Promise<any>;
++  signUp: (input: SignUpInput) => Promise<SignUpOutput>;
+```
+
+## Auth.resendSignUp
+
+```diff
+-  import { Auth } from 'aws-amplify';
++  import { resendSignUpCode } from 'aws-amplify/auth';
+
+-  Auth.resendSignUp: (username: string, clientMetadata?: ClientMetaData) => Promise<any>;
++  resendSignUpCode: (input: ResendSignUpCodeInput) => Promise<ResendSignUpCodeOutput>;
+```
+
+## Auth.signIn
+
+```diff
+-  import { Auth } from 'aws-amplify';
++  import { signIn } from 'aws-amplify/auth';
+
+-  Auth.signIn: (usernameOrSignInOpts: string | SignInOpts, pw?: string, clientMetadata?: ClientMetaData) => Promise<CognitoUser | any>;
++  signIn: (input: SignInInput) => Promise<SignInOutput>;
+```
+
+## Auth.federatedSignIn
```

Comment:
I think it is worth to mention that the usage of `federate with identity pools` using this API is deprecated and pointing customers to this guide instead: https://docs.amplify.aws/react/build-a-backend/auth/advanced-workflows/#identity-pool-federation